### DISCUSSION TITLE
Ciallo～(∠・ω< )⌒★

### DISCUSSION THREAD

**Discussion by Gwiazda-0** [share121/inter-knot#3379] [其他]
Ciallo～(∠・ω< )⌒★

**Comment by yui-hanasaka** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >Ciallo~（∠・ω< ）⌒ ★

**Comment by Alphooo422** [comment]
你们柚子厨真的是
